deploy/hubble/manifests/controller/helm/retina/templates/agent/configmap.yaml

```diff
@@ -131,8 +131,8 @@ data:
     enabledPlugin: {{ .Values.enabledPlugin_win }}
     metricsInterval: {{ .Values.metricsInterval }}
     metricsIntervalDuration: {{ .Values.metricsIntervalDuration }}
-    enableTelemetry: {{ .Values.enableTelemetry }}
-    enablePodLevel: {{ .Values.enablePodLevel }}
+    enableTelemetry: false
+    enablePodLevel: false
```

Comment:
if this isn't supported on windows at all, hardcoding false in this windows configmap makes sense